cmd/podman/create.go

```diff
@@ -669,3 +702,14 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 	}
 	return config, nil
 }
+
+//CreatePortBinding takes port (int) and IP (string) and creates an array of portbinding structs
+func CreatePortBinding(hostPort int, hostIP string) []nat.PortBinding {
+	pb := nat.PortBinding{
+		HostPort: strconv.Itoa(hostPort),
+	}
```

Comment:
Why not just do 
HostIP: hostIP

Then eliminate the check below.  By default HostIP will be "", and if hostIP="" nothing will change and code is nicer.

Reply:
fixed